src/lib.rs

```diff
@@ -1,6 +1,6 @@
 //! Client library for the [Discord](https://discordapp.com) API.
 //!
-//! The Discord API can be divided into three main components: the RESTful API
+//! The Discord API can be divided into three main components: the `RESTful` API
```

Comment:
Clippy is often incorrect here. It's meant to notify about structs/modules which should be ticked for highlighting, but in this case un-ticked is correct.

Relevant lint: <https://github.com/Manishearth/rust-clippy/wiki#doc_markdown>